meteor-app/imports/ui/assessment/assessment-job-card.js

```diff
@@ -1,58 +1,106 @@
+import printJobCart from '/imports/ui/assessment/assessment-print-job'
 
 class JobCard extends Component {
 
+  updateButton = () => {
+    const jobId = this.props.currentJob._id
+    const status = this.props.currentJob.status
+    const updatedStatus = status+1
 
-    render() {
-        const { status, bikeDetails, services, pickupDate, totalCost } = this.props.currentJob
-        const make = bikeDetails.make
-        const model = bikeDetails.model
-        const color = bikeDetails.color
-        const serviceLevel = services.baseService
-        const pickUp = pickupDate.toLocaleDateString()
-        return (
-            <Card>
-                <Container className="job-card-container" >
-                    <Card.Header style={{ textAlign: "Center", fontSize: "1.5em", margin: "10px" }} >
-                        {`Job Status: ${CONSTANTS.ORDER_STATUS_READABLE[status]}`}
-                    </Card.Header>
-                    <Card.Content style={{ textAlign: "Center", fontSize: "1em" }}>
-                        <List>
-                            <List.Item>{`Make: ${make}`}</List.Item>
-                            <List.Item>{`Model: ${model}`}</List.Item>
-                            <List.Item>{`Color: ${color}`}</List.Item>
-                            <List.Item>{`Service Level: ${serviceLevel}`}</List.Item>
-                            <List.Item>{`Pick-Up Date: ${pickUp}`}</List.Item>
-                            <List.Item>{`Total Price: $${totalCost/100}`}</List.Item>
-                        </List>
-                    </Card.Content>
-                    <Container style={{ textAlign: "Center" }}>
-                        <Button.Group style={{  width: "80%" }} vertical>
-                            <Button className="positive ui button"
-                            style={{ marginTop: '5px', marginBottom: '5px' }} >
-                                <h2>Start Job</h2>
-                            </Button>
-                            <Button className="positive ui button"
-                            style={{ marginTop: '5px', marginBottom: '5px' }} >
-                                <h2>Print Job Card</h2>
-                            </Button>
-                            <Button className="negative ui button"
-                            style={{ marginTop: '5px', marginBottom: '5px' }} >
-                                <h2>Cancel Job</h2>
-                            </Button>
-                        </Button.Group>
-                    </Container>
-                </Container>
-            </Card>
+    try {
+      if (status >= 5) return
+      Meteor.call('assessment.updateJobStatus', jobId, updatedStatus)
+    } catch (error) {
+      throw new Meteor.Error(error)
+    }
+  }
 
-    )}
-}
+  cancelButton = () => {
+    const jobId = this.props.currentJob._id
+    const status = this.props.currentJob.status
+    const cancelStatus = 6
+    const reopenStatus = 1
+    try {
+      if (status < 5) {
```

Comment:
Again you are assuming the number 5 has some magic properties. Please import and use constants from /imports/api/constants.js